tailwind.config.js

```diff
@@ -43,22 +44,36 @@ export default {
       textColor: {
         default: 'var(--foreground-default, #000)',
         dimmer: 'var(--foreground-dimmer, #555)',
-        dimmest: 'var(--foreground-dimmest, #b5b5b5)'
-      },
-      // 基础颜色
-      colors: {
-        // 危险颜色
-        'negative-dimmest': 'var(--nagative-dimmest, #eddcdc)',
-        'negative-default': 'var(--negative-default, #bb4c51)',
-        // 成功颜色
-        'positive-dimmer': 'var(--positive-dimmer, #37733c)',
-        'positive-default': 'var(--positive-default, #4e9553)',
-        'positive-higher': 'var(--positive-higher, #cff1d8)',
-        'positive-highest': 'var(--positive-highest, #d5e4d9)',
-        // 标准颜色
-        'primary-dimmest': 'var(--primary-dimmest, #d6e8ff)',
-        'primary-default': 'var(--primary-default, #1c74dd)'
+        dimmest: 'var(--foreground-dimmest, #b5b5b5)',
+        // 复杂文字高亮
+        'white-default': 'var(--accent-white-default, #fff)'
       }
+    },
+    // 基础颜色
+    colors: {
```

Comment:
I have covered the default colors of tailwindcss, and you need to delete these colors in the table section  @Feudalman